Marlin/src/inc/Version.h

```diff
@@ -69,13 +69,6 @@
   #define PROTOCOL_VERSION "1.0"
 #endif
 
-/**
- * Define a generic printer name to be output to the LCD after booting Marlin.
- */
-#ifndef MACHINE_NAME
-  #define MACHINE_NAME "3D Printer"
-#endif
-
```

Comment:
`MACHINE_NAME` shouldn't be removed from this file since it's used by vendors to set a machine name (among many other defaults).